docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
       - "listen_addresses=*"
     restart: always
   redis:
-    image: redis:latest@sha256:b4e56cd71c74e379198b66b3db4dc415f42e8151a18da68d1b61f55fcc7af3e0
+    image: redis:latest@sha256:c95835a74c37b3a784fb55f7b2c211bd20c650d5e55dae422c3caa9c01eb39fa
```

Comment:
@jimleroyer @patheard is there any concern here that we could be jumping major redis versions given we have it set as `redis:latest`? 

I can't figure out based on the SHA what this upgrade is.  Is there a listing somewhere where you can see what version redis was on the old SHA vs the new one?

Reply:
Good catch @andrewleith - I'd recommend pinning this to the same Redis version you're running in staging/prod rather than letting it move with `latest`.

The easiest fix would be to close this PR and open a new one pinning the version to the tag and then let a future Renovate PR pin the tag to a SHA.

Reply:
The 6.2.6 version is what we have in AWS Elasticache, but we could set it to latest patch of 6.2.* if you want 👍 (as AWS should automatically upgrade to latest patch anyway).

Reply:
Done: https://github.com/cds-snc/notification-api/pull/1656